model/converter/thrift/zipkin/fixtures/jaeger_01.json

```diff
@@ -61,7 +61,7 @@
       }
     },
     {
-      "traceID": "1",
+      "traceID": "20000000000000001",
```

Comment:
How can be this split into `lo` and `hi`? 

https://github.com/uber/jaeger/blob/5e5a361babba385bd0b9868bace1e10c75fc0829/model/span.go#L55
https://github.com/uber/jaeger/blob/5e5a361babba385bd0b9868bace1e10c75fc0829/model/span.go#L42-L44